UKHomeOffice/cerberus-service
# COP-10460: Add tests for checking Baggage details of Airpax task on task details page

## Description
Add tests for checking Baggage details of Airpax task on task details page

## To Test
npm run cypress:runner

run tests from spec `airpax-task-details.spec.js` 

`Should verify Baggage details of an AirPax task with Single Passenger on task details page`
`Should verify Baggage details of an AirPax task with Multiple Passenger on task details page`

## Developer Checklist



\* Required
